src/core/system.conf.in

```diff
@@ -53,7 +53,7 @@
 #DefaultLimitSTACK=
 #DefaultLimitCORE=
 #DefaultLimitRSS=
-#DefaultLimitNOFILE=
+#DefaultLimitNOFILE=@HIGH_RLIMIT_NOFILE@
```

Comment:
hmm, actually, this should be `DefaultLimitNOFILE=1024:@HIGH_RLIMIT_NOFILE@`

Reply:
Updated.